huntflow_api_client/models/request/vacancy_requests.py

```diff
@@ -0,0 +1,43 @@
+import typing as t
+
+from pydantic import BaseModel, EmailStr, Extra, Field, PositiveInt
+
+from huntflow_api_client.models.common import JsonRequestModel
+
+
+class VacancyRequestAttendee(BaseModel):
```

Comment:
it's not used at all, but there must be attendees in CreateVacancyRequestRequest. https://master-api.huntflow.dev/v2/docs#post-/accounts/-account_id-/vacancy_requests